qdrant-landing/content/articles/binary-quantization.md

```diff
@@ -0,0 +1,236 @@
+---
+title: "Binary Quantization - Vector Search, 40x Faster "
+short_description: "Binary Quantization is a newly introduced mechanism of reducing the memory footprint and increasing performance"
+description: "Binary Quantization is a newly introduced mechanism of reducing the memory footprint and increasing performance"
+social_preview_image: /articles_data/binary-quantization/social_preview.png
+small_preview_image: /articles_data/binary-quantization/binary-quantization-icon.svg
+preview_dir: /articles_data/binary-quantization/preview
+weight: -40
+author: Nirant Kasliwal
+author_link: 
+date: 2023-09-18T13:00:00+03:00
+draft: false
+keywords:
+  - vector search
+  - binary quantization
+  - memory optimization
+---
+
+#### Optimizing high-dimensional vectors 
+
+Qdrant is built to handle typical scaling challenges: high throughput, low latency and efficient indexing. **Binary quantization (BQ)** is our latest attempt to give our customers the edge they need to scale efficiently. This feature is particularly excellent for collections with large vector lengths and a large number of points. 
+
+Our results are dramatic: Using BQ will reduce your memory consumption and improve retrieval speeds by up to 40x.
+
+As is the case with other quantization methods, these benefits come at the cost of recall degradation. However, our implementation lets you balance the tradeoff between speed and recall accuracy at time of search, rather than time of index creation. 
+
+The rest of this article will cover:
+1. The importance of binary quantization
+2. Basic implementation using our Python client
+3. Benchmark analysis and usage recommendations
+
+![Speed vs recall of quantization and indexing methods](/articles_data/binary-quantization/bq-1.png)
+
+## What is Binary Quantization?
+Binary quantization (BQ) converts any vector embedding of floating point numbers into a vector of binary or boolean values. This feature is an extension of our past work on [scalar quantization](/articles/scalar-quantization/) where we convert `float32` to `uint8` and then leverage a specific SIMD CPU instruction to perform fast vector comparison. 
+
+![What is binary quantization](/articles_data/binary-quantization/bq-2.png)
+
+**This binarization function is how we convert a range to binary values. All numbers greater than zero are marked as 1. If it's zero or less, they become 0.**
+
+The benefit of reducing the vector embeddings to binary values is that boolean operations are very fast and need significantly less CPU instructions. In exchange for reducing our 32 bit embeddings to 1 bit embeddings we can see up to a 40x retrieval speed up gain!
+
+One of the reasons vector search still works with such a high compression rate is that these large vectors are over-parameterized for retrieval. This is because they are designed for ranking, clustering, and similar use cases, which typically need more information encoded in the vector. 
```

Comment:
I'm not a pride guy and can say directly: I cannot understand this couple of sentences. I see that it's from the scientific article with a specified lexic. QDrant is against academics!

Reply:
Can you please explain it simply even if more words are required?

Reply:
tried as much as possible